packages/api/tests/lambdas/test-sf-semaphore-down.js

```diff
@@ -37,6 +40,43 @@ const createCloudwatchEventMessage = ({
   }
 });
 
+const createCloudwatchPackagedEventMessage = ({
+  status,
+  queueName,
+  source = sfEventSource
+}) => ({
+  source,
+  detail: {
+    status,
+    output: JSON.stringify({
+      cumulus_meta: {
+        execution_name: randomString(),
+        queueName
+      },
+      replace: {
+        Bucket: 'cumulus-sandbox-testing',
+        Key: 'stubbedKey'
+      }
+    })
+  }
+});
+
+
+const packagedEventReturn = ((queueName) => (
```

Comment:
I would call this`createExecutionMessage`. You could use it in `createCloudwatchEventMessage` above as well to avoid repetition, but maybe that's bad for some reason i'm not thinking of

Reply:
👍

Reply:
380a681